index.html

```diff
@@ -1,302 +1,530 @@
-            <div>
-              <h1>Uber Navigation</h1>
-            </div>
-            <div class="lister">
-              <p><strong>CANOPY</strong></p>
-              <div class="point"></div>
-              <p>Back End Dev</p>
-              <div class="point"></div>
-              <p>2015</p>
+        <ul class="project-works row-reverse-desktop">
+          <li class="project-image-container">
+            <img
+              src="./images/SnapshootPortfolioD.svg"
+              alt="Tonic"
+              class="mobile-image"
+            />
+            <img
+              src="./images/SnapshootDesktopPortfolioD.svg"
+              alt="Tonic"
+              class="desktop-image"
+            />
+          </li>
+          <li class="project-body-block">
+            <h2 class="project-title hide-for-desktop">Multi-Post Stories</h2>
+            <h2 class="project-title hide-for-mobile">Uber Navigation</h2>
+            <div class="project-info">
+              <span class="project-info-items canopy hide-for-desktop"
+                >CANOPY</span
+              >
+              <span class="project-info-items canopy hide-for-mobile"
+                >FACEBOOK</span
+              >
+              <img src="./images/Counter.svg" alt="counter" />
+              <span class="project-info-items hide-for-desktop"
+                >Back End Dev</span
+              >
+              <span class="project-info-items hide-for-mobile dsk-fw"
+                >Lead Developer</span
+              >
+              <img src="./images/Counter.svg" alt="counter" />
+              <span class="project-info-items hide-for-desktop">2015</span>
+              <span class="project-info-items hide-for-mobile dsk-fw"
+                >2018</span
+              >
             </div>
+            <p class="project-primary-text hide-for-desktop">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
 
-            <p class="para1">
+            <p class="project-primary-text hide-for-mobile">
               A smart assistant to make driving more safe, efficient, and fun by
               unlocking your most expensive computer: your car.
             </p>
-            <ul>
-              <li class="html"><a href="#"> html</a></li>
-              <li class="css"><a href="#">css</a></li>
-              <li class="javaScript"><a href="#">javaScript</a></li>
+            <ul class="tags">
+              <li class="tag">html</li>
+              <li class="tag hide-for-mobile">Ruby on rails</li>
+              <li class="tag">css</li>
+              <li class="tag">javaScript</li>
             </ul>
-            <div class="to-project"><a href="#">See Project</a></div>
-          </div>
-        </div>
-      </div>
+            <button class="button see-project" type="button">
+              See Project
+            </button>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] Following the project requirements, The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. Checkout the requirements below 👇 :

<img width="505" alt="image" src="https://user-images.githubusercontent.com/77190203/157649553-844d7570-0411-4439-a7d8-7ddd9a4de306.png">